### PR TITLE
Compatible version for php5 and php7

### PR DESCRIPTION
**Pull request by razbakov**
Waiting for [coding-standards:0.1.3](https://github.com/banovo/coding-standards/milestone/3)
Release as 0.3.0

This branch is based on version `0.1.*`, it includes bin/checker. That was a requirement. We used branch `0.2.*` for php7 apps. The problem was dependencies in composer. I solved it by adding Symfony ruleset to thia repository and by excluding dependencies we don't need now. 

This version works good on coma and orocrm.

Related [ch645]
Closes [ch640]

### CODE REVIEW

**Review by eFzvai — CHANGES_REQUESTED** [review]
The ticket [ch640]() required the coding style check binary for php7. But this pull request only add fix some test. 
This pull request can not close the ticket [ch640]

**Comment by razbakov** [comment]
This branch is based on version `0.1.*`, it includes bin/checker. That was a requirement. We used branch `0.2.*` for php7 apps. The problem was dependencies in composer. I solved it by adding Symfony ruleset to thia repository and by excluding dependencies we don't need now. 

This version works good on coma and orocrm.

**Comment by eFzvai** [comment]
Ok, but why this information is not in the commit message?

File: README.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
 [![Build Status](https://travis-ci.org/banovo/coding-standards.svg?branch=master)](https://travis-ci.org/banovo/coding-standards)
 
 ## Installation
 
-### Composer
+Add to `composer.json`:
+
+```json
+"require-dev": {
+    "banovo/coding-standard": "0.1.*"
```
Comment (eFzvai) [inline README.md]: `0.3.*`

**Review by eFzvai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (razbakov) [reply README.md]: fixed

**Comment by razbakov** [comment]
Updated PR description, which will be used as commit message.

**Review by eFzvai — DISMISSED** [review]

**Review by eFzvai — APPROVED** [review]

**Comment by razbakov** [comment]
Recreated #16

**Pull request closed without merging**